packages/shared/src/components/Feed.tsx

```diff
@@ -424,6 +424,7 @@ export default function Feed<T>({
           onPreviousPost={onPrevious}
           onNextPost={onNext}
           postPosition={postPosition}
+          postSource={selectedPost.source}
```

Comment:
passing down the source from the feed so we can use it to determine the modal loading layout

Reply:
I think we should pass the whole post instead. From there we will use the `source` property. We might even be able to leverage the `post.type` on some cases.